lib/github_backend.rb

```diff
@@ -172,7 +49,7 @@ def issue_count_by_status(opts)
                                              value: 1)
         end
       rescue Octokit::Error => exception
-        Raven.capture_exception(exception)
+        @logger.debug("Octokit exception: #{exception}")
```

Comment:
Since there's an `attr_accessor` defined for `logger`, shouldn't we use `logger` instead of `@logger`?